site/source/tutorials/007_theming/index.md

```diff
@@ -34,43 +34,47 @@ In order to theme our widgets, we must ensure that they each extent `ThemeableMi
 If you cannot see the application, remember to run `dojo build -w` to build the application and start the development server.
 {% endaside %}
 
-This `Banner` widget will now have access to the class files in `banner.m.css` and can receive a `theme`. We use the `root` class here as detailed above to ensure that the theme we create can easily target the correct node.
+This `Banner` widget will now have access to the classes in `banner.m.css` and can receive a `theme`. We use the `root` class to ensure that the theme we create can easily target the correct node.
 
 {% instruction 'Create a new css file for `Banner` named `banner.m.css`.' %}
 
-We will create an empty `root` class for now as our base theme does not require any styles to be added to the `banner`.
+We will create an empty `root` class for now as our base theme does not require any styles to be added to the `Banner` widget.
 
 {% include_codefile 'demo/finished/biz-e-corp/src/styles/banner.m.css' lang:css %}
 
-{% instruction 'Now, lets look at changing `WorkerForm`' %}
+{% instruction 'Now, let's look at changing `WorkerForm`' %}
 
-`WorkerForm` already uses the `ThemeableMixin` and has a `workerForm` class on it's root node. Lets go ahead and change that to a `root` class, but while we're there, we will create a `rootFixed` class too and apply that as a `fixed` class to the root node. This passes a classname to the theming system that cannot be changed or overridden via a theme and ensures that structured or nested styles are not lost when a theme is used.
+{% aside 'Fixed Classes' %}
+Fixed classes apply styles that _cannot_ be overridden by a theme, using a suffix is a convention that helps differentiate the intention of these classes easily.
```

Comment:
helps differentiate the intention of these classes easily -> helps easily differentiate the intention of these classes